docs/sources/helm-charts/mimir-distributed/_index.md

```diff
@@ -8,15 +8,18 @@ keywords:
   - Grafana Enterprise Metrics
   - Grafana metrics
 cascade:
-  # Used to generate relative URL paths to related documentation
   mimir_docs_version: "v2.9.x"
   gem_docs_version: "v2.9.x"
```

Comment:
Let's rename to GEM_DOCS_VERSION if we rename the mimir version - needs update in operations/helm/charts/mimir-distributed/RELEASE.md as well